src/AWS.Deploy.CLI/Commands/DeleteDeploymentCommand.cs

```diff
@@ -65,6 +74,18 @@ public async Task ExecuteAsync(string stackName)
                 // Monitor updates the stdout with current status of the CloudFormation stack
                 var _ = monitor.StartAsync();
 
+                // delete the ECR repository if exists for a container based deployment
+                // repository must be tagged with deploy tool tag
+                var repository = await GetECRRepository(stackName.ToLower());
```

Comment:
This is potentially deleting customer data. We don't know if they have reused this repository for other purposes or even deployed the image we created for another deployment.

This should be an opt-in parameter to delete the repository to be safe from unintentionally deleting customer data. If they don't opt in we could put a warning messages to let them know this resource still exists.